resources/views/auth/register/steps/4.blade.php

```diff
@@ -6,15 +6,15 @@
     <!-- Password -->
     <div class="field @error('password') field--error @enderror stack">
         <x-hearth-label for="password" :value="__('hearth::auth.label_password')" />
-        <x-hearth-hint for="password">
+        <div class="field__hint" id="password-hint">
             {{ __('For your security, please make sure your password has:') }}
```

Comment:
Wrap this in a `<p>` tag as we aren't processing it with Markdown as is normally the case in the hint component.